java/datastructures/lib/src/test/java/datastructures/hashmaptest/hashMapTest.java

```diff
@@ -0,0 +1,54 @@
+package datastructures.hashmap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.*;
```

Comment:
Be careful with your imports here. Because you're importing all subclasses of `java.util`, you're also importing (and therefore testing) Java's built-in HashMap class instead of your own implementation.